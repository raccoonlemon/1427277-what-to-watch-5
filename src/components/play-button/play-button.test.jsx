import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import PlayButton from "./play-button";

const renderer = new ShallowRenderer();

it(`<PlayButton> renders correctly`, () => {
  renderer.render(<PlayButton id={`1`}/>);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
