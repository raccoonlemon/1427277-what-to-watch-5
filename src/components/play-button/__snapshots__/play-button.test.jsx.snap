// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayButton> renders correctly 1`] = `
<Link
  className="btn btn--play movie-card__button"
  to="/player/1"
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</Link>
`;
